cmd/weaver-kube/deploy.go

```diff
@@ -37,12 +37,26 @@ const (
 )
 
 var (
-	deployCmd = tool.Command{
+	deployFlags       = flag.NewFlagSet("deploy", flag.ContinueOnError)
+	dockerhubUsername = deployFlags.String("username", "", "Docker Hub username")
```

Comment:
Awesome, thanks for fixing!

Btw., since we now use the `docker` binary to push (as opposed to the Go Docker client libraries), is there any need/benefit for providing this flag. Suppose you're logged in using `docker login` as foo@gmail.com, but you pass username `bar@gmail.com`. Would `docker push` authenticate you correctly as `bar@gmail.com`?

I'm asking because we can perhaps just get rid of this flag and use whatever `docker login` username is active. Alternatively, we pass in the flag, but then we call all `docker` commands with --username flag attached to it (if docker allows this).

The latter above is how we handle gcloud commands.

Reply:
Great point! I removed the `--username` flag. I don't think `docker push` supports a `--username` flag.